samples/java-spring-eventsourced-customer-registry-subscriber/src/it/java/customer/api/CustomerIntegrationTest.java

```diff
@@ -61,8 +61,8 @@ public CustomerIntegrationTest(ApplicationContext applicationContext) {
     confMap.put("kalix.user-function-port", dockerComposeUtils.userFunctionPort());
     // don't kill the test JVM when terminating the KalixRunner
     confMap.put("kalix.system.akka.coordinated-shutdown.exit-jvm", "off");
-    // dev-mode should be false when running integration tests
-    confMap.put("kalix.dev-mode.enabled", false);
+    // FIXME: should point to docker-compose-integration.yml instead
+    conf.put("kalix.dev-mode.docker-compose-file", "none");
```

Comment:
Not sure I get this FIXME.. why should it point to "docker-compose-integration.yml" and what is preventing it?

Reply:
I forgot about this FIXME. 

I set it explicitly to `none` because I didn't want the KalixRunner to read the default. Which is, `docker-compose.yml`. 

Instead, we should configure it to use `docker-compose-integration.yml` and all the DockerComposeUtils that we are explicitly calling in this test should be removed. 

This test should start Klaix using `docker-compose-integration.yml` in dev-mode, run the assertions and let the containers be shutdown during coordinated shutdown.